helm-chart/binder-launches/templates/configmap.yaml

```diff
@@ -12,12 +12,8 @@ data:
     {{- if .Values.port }}
     port: {{ .Values.port }},
     {{- end }}
-    {{- if .Values.debug }}
     debug: {{ .Values.debug }},
```

Comment:
Can we keep this check for debug? It is no problem if debug is not set when it is false, because in the application it is false by default. Sorry, this is confusing, maybe we should a comment about it?

Reply:
yeah it would be nice with a comment and be more explicit.
(this seems like there should be a standard solution ¯\_(ツ)_/¯)